Source/Blazorise/Blazorise.csproj

```diff
@@ -23,4 +24,44 @@
     <EmbeddedResource Include="Resources\Localization\*.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="bundleconfig.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Scripts\blazorise.dependencies.js" />
+    <None Remove="Scripts\blazorise.functions.js" />
+    <None Remove="Scripts\blazorise.main.js" />
+    <None Remove="Scripts\blazorise.table.js" />
+    <None Remove="Scripts\blazorise.tooltip.js" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Scripts\blazorise.table.js">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+    <Content Include="Scripts\blazorise.functions.js">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+    <Content Include="Scripts\blazorise.tooltip.js">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+    <Content Include="Scripts\blazorise.main.js">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+    <Content Include="Scripts\blazorise.dependencies.js">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BuildBundlerMinifier" Version="3.2.449" />
```

Comment:
What happens with nuget packages? Are these assemblies included or excluded on dotnet pack? You can check by running dotenet-pack.cmd in NuGet folder

Reply:
Oh that's a good question, you're worried it's smthg that goes embbeded on the final Product, is that it?

Reply:
Hmm I don' see anything wrong with the packages. I will have to setup a local nuget server later to test... But i'm pretty sure, that be default is the standard assembly behaviour. It will list those new references as Blazorise dependencies.

Reply:
You only need to run the script and open .nuget file in any zip viewer to see the result.

Reply:
I did that. I don't see anything wrong with it. 
But it did pick up the scripts... that should be setting Copy to Ouput to false.
![image](https://user-images.githubusercontent.com/22283161/126907907-342d2503-6c90-4757-8ef3-b337de353308.png)

![image](https://user-images.githubusercontent.com/22283161/126907934-b3c42886-b987-4294-a48e-ad85801cd54c.png)

Reply:
Yeah, scripts should not be there